test/unit/view.js

```diff
@@ -151,101 +247,85 @@ describe('View class', () => {
           },
         };
         view.append(div);
+        assert.calledOnce(view.iframe.document.body.appendChild);
```

Comment:
For consistency with other tests, can we put the spy into it's own const and assert with that instead?